modules/stanford_profile_styles/webpack.config.js

```diff
@@ -46,6 +46,8 @@ var webpackConfig = {
   devtool: 'source-map',
   // What build?
   entry: {
+    "admin": path.resolve(__dirname, srcSass + "/admin/index.scss"),
+    "field_widgets": path.resolve(__dirname, srcSass + "/admin/stanford_profile_styles.field_widgets.scss"),
```

Comment:
@pookmish,
Is this what you're looking for?

I'm missing something as it's outputting to `/dist/css/field_widgets.css`

Reply:
The key in the entry object is what denotes the file name. 

Change: 
`"field_widgets": path.resolve(__dirname, srcSass + "/admin/stanford_profile_styles.field_widgets.scss")`

to:
`"stanford_profile_styles.field_widgets": path.resolve(__dirname, srcSass + "/admin/stanford_profile_styles.field_widgets.scss")`